src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -238,7 +238,7 @@ const VirtualizedMessageListWithContext = <
 
       const firstOfGroup =
         shouldGroupByUser &&
-        streamMessageIndex > 0 &&
+        message.user?.id === messageList[streamMessageIndex + 1]?.user?.id &&
```

Comment:
This will cause a couple of css bugs just so you are aware

Reply:
Hi guys! Thank you for reviewing this bug I've reported, you are doing a great job about it.

Just to clarify the current issue, I'm having this problem:
![image](https://user-images.githubusercontent.com/7905320/149898162-b5d2a1e0-9b4b-46a1-aa2e-a9d400a28941.png)

As you can see the first message in a chat is not showing as expected (No avatar, no user title,...). And that's because the `firstOfGroup` there is returning false, because it doesn't have a previous message.

I don't know if this solution will work, as you are checking that the next message exists too. So if a user puts a single message then `firstOfGroup` will be `false` as it doesn't have any messages next to it. For example, in the screenshot the last message will be not showing either the Avatar or the user title.

Reply:
Hi Ivan,

I've updated the PR to remove the index check to fix the first message in a channel. We don't need to check for a previous message because the [endOfGroup](https://github.com/GetStream/stream-chat-css/blob/bde73322aa0a8023be82b4e730b8d128161b4451/src/styles/VirtualMessage.scss#L62) will add a second class the messages like the last message above, which will include attributes like the avatar.